### PR TITLE
Natspec legacy contracts update

### PR DESCRIPTION
**Pull request by cyborgdennett**
**Description**

I added a couple of `@param` and `@return` where there were none, and added `@inheritdoc` where there was an interface contract available. I only took a look at legacy contracts.

### CODE REVIEW

**Review by maurelian — APPROVED** [review]
LGTM thank you!

File: packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol [hunk 1]
```diff
@@ -33,6 +33,9 @@ contract L2CrossDomainMessenger is IL2CrossDomainMessenger {
      * Constructor *
      ***************/
 
+     /**
```
Comment (smartcontracts) [inline packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol:36]: I think there's an extra space here

**Review by smartcontracts — COMMENTED** [review]
This is great, thanks! Dropped one quick comment

File: packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol (hunk 1 above)
Comment (cyborgdennett) [reply packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol:36]: Good catch! 👁️

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
CI will fail due to permission errors but this should be fine to merge since its just changing comments

**Pull request merged**